pkg/cmd/repo/create/create.go

```diff
@@ -35,6 +35,77 @@ type iprompter interface {
 	Confirm(string, bool) (bool, error)
 }
 
+type VisibilityOption string
+
+const (
+	VisibilityOptionPublic   VisibilityOption = "Public"
+	VisibilityOptionPrivate  VisibilityOption = "Private"
+	VisibilityOptionInternal VisibilityOption = "Internal"
+)
+
+func (v VisibilityOption) forSelectionPrompt() string {
+	return string(v)
+}
+
+func (v VisibilityOption) forConfirmationPrompt() string {
+	return strings.ToLower(string(v))
+}
+
+func (v VisibilityOption) toRepoVisibility() visibility {
+	switch v {
+	case VisibilityOptionPublic:
+		return visibilityPublic
+	case VisibilityOptionPrivate:
+		return visibilityPrivate
+	case VisibilityOptionInternal:
+		return visibilityInternal
+	default:
+		panic("invalid visibility option: " + v)
+	}
+}
```

Comment:
It's a reasonable question but I'm going to push back on it. The UI and the API are separate domains and it's useful to convey intent through the different types, and to allow them to evolve separately. If I decide I want to produce the UI selections in a different way, I don't need to touch the API types. 

In retrospect I probably should have made this a bare function instead of a method.